app/Filament/Resources/EventResource/RelationManagers/AchievementsRelationManager.php

```diff
@@ -79,7 +80,30 @@ public function table(Table $table): Table
 
             ])
             ->headerActions([
-
+                Tables\Actions\Action::make('add-achievements')
+                    ->label('Add additional achievements')
+                    ->modalHeading('Add additional achievements')
+                    ->form([
+                        Forms\Components\TextInput::make('numberOfAchievements')
+                            ->label('Number of achievements')
+                            ->numeric()
+                            ->default(1)
+                            ->required(),
+                    ])
+                    ->action(function (array $data): void {
+                        $numberOfAchievements = (int) $data['numberOfAchievements'];
+                        /** @var Event $event */
+                        $event = $this->getOwnerRecord();
+                        $user_id = $event->achievements->first()?->achievement->user_id ?? EventAchievement::RAEVENTS_USER_ID;
+
+                        (new AddAchievementsToEventAction())->execute($event, $numberOfAchievements, $user_id);
+
+                        Notification::make()
+                            ->title("Created $numberOfAchievements new " . Str::Plural('achievement', $numberOfAchievements))
```

Comment:
nit: `Plural` should have a lowercase "p".

Reply:
done